pkg/common/utils/mysql/mysql.go

```diff
@@ -0,0 +1,46 @@
+package mysql
+
+import (
+	"database/sql"
+	"fmt"
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/jmoiron/sqlx"
+	"k8s.io/klog/v2"
+)
+
+type DBConfig struct {
+	User     string
+	Password string
+	Host     string
+	Port     string
+	Database string
+}
+
+type DB struct {
+	*sqlx.DB
+}
+
+func NewDorisSqlDB(cfg DBConfig) (*DB, error) {
+	dsn := fmt.Sprintf("%s:%s@tcp(%s:%s)/%s", cfg.User, cfg.Password, cfg.Host, cfg.Port, cfg.Database)
+	db, err := sqlx.Open("mysql", dsn)
+	if err != nil {
+		klog.Errorf("failed open doris sql client connection, err: %s \n", err)
+		return nil, err
+	}
+
+	if err = db.Ping(); err != nil {
```

Comment:
ping is not required when opening a connection have the ability of ping.  decommission is not a frequently operation.  keeping a connection is not necessary.